test/controllers/api/v2/content_view_filters_controller_test.rb

```diff
@@ -41,13 +41,33 @@ def setup
       end
     end
 
-    def test_index
+    def test_index_view
```

Comment:
Nitpick: `test_index_with_content_view`  as the word 'view' confused me with the Rails term